lib/screens/authentication/recover/recover_account_timer/interactor/viewmodels/recover_account_timer_state.dart

```diff
@@ -3,18 +3,20 @@ part of 'recover_account_timer_bloc.dart';
 class RecoverAccountTimerState extends Equatable {
   final PageState pageState;
   final ActiveRecoveryModel recoveryModel;
+  final GuardiansConfigModel configModel;
   final CurrentRemainingTime? currentRemainingTime;
   final PageCommand? pageCommand;
```

Comment:
renamed this variable since it's actually a fixed point in time, not a relative value in seconds